### DISCUSSION TITLE
Running Workspace Tests in a Monorepo appears to Force Mode "Test" for projects

### DISCUSSION THREAD

**Discussion by thuringia** [vitest-dev/vitest#6789] [Q&A]
Regardless of which value is passed as `--mode` to `vitest` on the top-level of the monorepo, each project gets launched with a mode of `test`.
This breaks our `vite` config because it relies on the mode to configure plugins for code generation and validation of configuration. Is this intentional or a bug?

**Comment by sheremet-va** [comment]
Seems like a bug, the mode should be inherited. Please create an issue with a minimal reproduction
